packages/features/form-builder/FormBuilderField.tsx

```diff
@@ -53,9 +54,20 @@ export const FormBuilderField = ({
 }) => {
   const { t } = useLocale();
   const { control, formState } = useFormContext();
+  const searchParams = useRouterQuery();
 
   const { hidden, placeholder, label } = getAndUpdateNormalizedValues(field, t);
 
+  const isPreFilledInUrl = (): boolean => {
```

Comment:
```suggestion
  const shouldBeDisabledDueToPrefill = (): boolean => {
```

This seems like a better name as it isn't just checking if the field is prefilled, it is also checking if the field is marked for disabling on prefill.